plugins/org.jboss.tools.openshift.client/.project

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why remove this from repo? We'll end up with the same problems as we faced a few weeks ago. Please put this back into the repo.